flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/reader/MySqlSourceReader.java

```diff
@@ -203,7 +203,8 @@ private MySqlBinlogSplit discoverTableSchemasForBinlogSplit(MySqlBinlogSplit spl
         final String splitId = split.splitId();
         if (split.getTableSchemas().isEmpty()) {
             try (MySqlConnection jdbc =
-                    DebeziumUtils.createMySqlConnection(sourceConfig.getDbzConfiguration())) {
+                    DebeziumUtils.createMySqlConnection(
+                            sourceConfig.getDbzConfiguration(), sourceConfig.getJdbcProperties())) {
```

Comment:
Why not expose a method `createMySqlConnection(MySqlSourceConfig sourceConfig)` ？